### ISSUE TITLE
WxAgg NavigationToolbar2 icons are clipped in wxpython 4.1

### ISSUE THREAD

**Issue by jbhopkins** [matplotlib/matplotlib#17326]
### Bug report

**Bug summary**
When using the wxagg backend, the NavigationToolbar2WxAgg has clipped icons with wxpython 4.1.0 (newest release). This happens with both the construction below and using pyplot when setting ```mpl.use('wxAgg')```. I have observed this on MacOS, it may also be true on Linux and Windows. This is most likely a result of wxpython 4.1.0 being based on wxwidgets 3.1 (previous versions were wxwidgets 3.0)

**Code for reproduction**
```python
import wx
import matplotlib as mpl
from matplotlib.backends.backend_wxagg import NavigationToolbar2WxAgg
from matplotlib.backends.backend_wxagg import FigureCanvasWxAgg

class PlotFrame(wx.Frame):
    def __init__(self, parent, title, *args, **kwargs):
        wx.Frame.__init__(self, parent, title=title, *args, **kwargs)

        self.fig = mpl.figure.Figure((5,4))
        self.subplot1 = self.fig.add_subplot(111)
        self.canvas = FigureCanvasWxAgg(self, wx.ID_ANY, self.fig)

        self.toolbar = NavigationToolbar2WxAgg(self.canvas)
        self.toolbar.Realize()

        sizer = wx.BoxSizer(wx.VERTICAL)
        sizer.Add(self.canvas, 1, wx.LEFT | wx.TOP | wx.GROW)
        sizer.Add(self.toolbar, 0, wx.GROW)

        self.SetSizer(sizer)
        self.Layout()
        self.Fit()
        self.CenterOnScreen()
        self.Show(True)

if __name__ == '__main__':
    app = wx.App(False)
    frame = PlotFrame(None, 'My Plot Frame')
    app.MainLoop()
```

**Actual outcome**
Clipped icons:
<img width="499" alt="mpl_toolbar_wx4p10" src="https://user-images.githubusercontent.com/23534688/81007346-ac568f00-8e16-11ea-8292-e0f12e312a9f.png">

**Expected outcome**
As with previous versions of wx, the toolbar icons should be fully visible (note that the MacOS toolbar background color has changed in wxpython 4.10, that seems to be expected):
<img width="497" alt="mac_mpl_toolbar_old" src="https://user-images.githubusercontent.com/23534688/81007475-e0ca4b00-8e16-11ea-8664-52df9beba29a.png">

**Matplotlib version**
  * Operating system: MacOS 10.14.6
  * Matplotlib version: 3.2.1.post2298+gd541b6dbb
  * Matplotlib backend (`print(matplotlib.get_backend())`): wxagg
  * Python version: 3.7.7
  * Other libraries:  wxpython 4.1.0

Python and dependencies installed via conda. Matplotlib installed from source from the latest version on github. Wxpython installed via pip.

**Comment by jbhopkins** [comment]
This appears to not be an issue in Windows, as shown below. I don't have access to a linux box with wxpython 4.1 (the build failed on my usual test machine and there don't seem to be any wheels available), so I can't test there.

Windows with wxpython 4.1:
![win10_mpl_toolbar_wx4p1](https://user-images.githubusercontent.com/23534688/81009621-9945be00-8e1a-11ea-9a39-959e80c20802.png)

**Comment by QuLogic** [comment]
Seems like this was supposed to be fixed by #17328.
